libnetwork/diagnostic/server.go

```diff
@@ -108,7 +108,7 @@ func (s *Server) DisableDiagnostic() {
 	s.Lock()
 	defer s.Unlock()
 
-	s.srv.Shutdown(context.Background())
+	s.srv.Shutdown(context.Background()) // nolint:errcheck
```

Comment:
Note: `nolint` comments should not have a leading whitespace (comments for automation don't have a leading whitespace)

Probably slightly more cleaner to use `_` to ignore the error than the `//nolint` comments